tests/test_with_mocked_bin_path.cpp

```diff
@@ -18,36 +18,23 @@
 #include "test_with_mocked_bin_path.h"
 #include "path.h"
 
-#include <cstdlib>
-
 namespace mp = multipass;
 namespace mpt = multipass::test;
 using namespace testing;
 
-namespace
-{
-void set_path(const std::string& value)
-{
-    auto ret = setenv("PATH", value.c_str(), 1);
-    if (ret != 0)
-    {
-        std::string message{"unable to modify PATH env variable err:"};
-        message.append(std::to_string(ret));
-        throw std::runtime_error(message);
-    }
-}
-} // namespace
-
 void mpt::TestWithMockedBinPath::SetUp()
 {
-    old_path = getenv("PATH");
-    std::string new_path{mpt::mock_bin_path()};
-    new_path.append(":");
-    new_path.append(old_path);
-    set_path(new_path);
+#ifdef WIN32
```

Comment:
Let's avoid platform ifdef's. You could use QDir::listSeparator() here instead.

Reply:
Darn, it was right there under my nose. Thanks!

Reply:
Cool, thanks!